src/pages/hackathon/redacted.tsx

```
@@ -0,0 +1,238 @@
+            <AccordionItem
+              key={f.question}
+              value={f.question}
+              className="my-4 rounded-lg border shadow-md"
+            >
+              <AccordionTrigger className="rounded px-4 py-3 text-left font-normal text-slate-500 data-[state=open]:bg-black/5 hover:bg-black/5 hover:no-underline focus:no-underline">
+                <span className="flex-1 text-left text-sm sm:text-base">
+                  {f.question}
+                </span>
+              </AccordionTrigger>
+              <AccordionContent className="px-4 pt-3 text-sm text-slate-700 sm:text-base [&_a]:text-blue-700">
+                <div dangerouslySetInnerHTML={{ __html: f.answer }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)